src/content/content_recommender.py

```diff
@@ -38,6 +39,13 @@ def test(self, member_id, potential_events, info_repo, simscores):
         for i in xrange(len(potential_events)):
             simscores[member_id][potential_events[i]] = similarity_scores[i]
 
+        top_indices = similarity_scores.argsort()[:-5:-1]
+        top_5_recommended_events = []
+        for i in top_indices:
+            top_5_recommended_events.append(potential_events[i])
+
+        top_5_recommendation_measurement(top_5_recommended_events, info_repo["members_events"][member_id], \
```

Comment:
You mean just send the second half of the partition as the arguments instead of info_repo["members_events"][member_id]?